test/functionalTest/cases/1308_subscription_cache/subCacheMixedSubscriptionsWithUpdatesAndDeletes.test

```diff
@@ -422,12 +424,12 @@ Date: REGEX(.*)
     "jsonRequests": "5",
     "measuring_interval_in_secs": "REGEX(\d+)",
     "statisticsRequests": "5",
-    "subCache": "REGEX([0-9a-f]{24})",
-    "subCacheInserts": "REGEX((1|2))",
-    "subCacheItems": "REGEX((1|2))",
+    "subCache": "REGEX([0-9a-f, ]{76})",
+    "subCacheInserts": "REGEX((5|8))",
+    "subCacheItems": "3",
```

Comment:
Yes, I know ...
I haven't even run trhis under valgrind yet.
There will be a few errors, without a doubt.
As I said, this PR is a little premature.
Will be fixed